clang_format/run_clang_format.sh

```diff
@@ -7,6 +7,17 @@ set -ue
 format_all() {
     CLANG_FORMAT_CONFIG=$(realpath $1)
 
+    cd $BUILD_WORKSPACE_DIRECTORY
+    if ! test -f .clang-format; then
+        echo ".clang-format file not found. Bazel will copy the default .clang-format file."
+        cp $CLANG_FORMAT_CONFIG .
+    fi
+    git ls-files '*.[ch]' '*.cpp' '*.cc' '*.hpp' | xargs -r $CLANG_FORMAT_BIN -i
```

Comment:
Should we have `*.cxx` `*.hxx` for completeness?

Reply:
Wouldn't hurt!